work-scheduler-data/src/main/java/com/github/gaalgergo/workscheduler/data/entities/VacationRequestEntity.java

```diff
@@ -0,0 +1,16 @@
+package com.github.gaalgergo.workscheduler.data.entities;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import java.time.LocalDate;
+
+@Entity
+@Data
+public class VacationRequestEntity {
+
+    private EmployeeEntity employeeEntity;
```

Comment:
Define the JPA association between `EmployeeEntity` and `VacationRequestEntity`.